clang/unittests/Format/FormatTestComments.cpp

```diff
@@ -3102,7 +3102,9 @@ TEST_F(FormatTestComments, DontAlignNamespaceComments) {
   StringRef Input = "namespace A {\n"
                     "  TESTSUITE(B) {\n"
                     "    namespace C {\n"
-                    "      namespace D {} // namespace D\n"
+                    "      namespace D {\n"
+                    "        // ...\n"
```

Comment:
```suggestion
                    "      namespace D { //\n"
```
To keep the `namespace` empty like before.